client/stylesheets/_toCat.scss

```diff
@@ -3398,11 +3400,15 @@ On click the images will be displayed at normal size to complete the effect
 .listingItemUser > ul {
     display: inline-block;
     margin: 0;
-    margin-top: 10px !important;
+}
+
+.listingItemUserImg {
+	margin-top: 10px !important;
 }
 
 .listingItemUserInfo {
     margin-left: 15px !important;
+    margin-top: 10px !important;
```

Comment:
!important should not be used<br>Line should be indented 2 spaces, but was indented 4 spaces